arrayProcessingTool.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="assets/styles/arrayProcessingTool.css">
+    <title>Title</title>
+</head>
+<body>
+<span>Enter array in '2 3 4...' format</span>
+<input id="array">
+<br/>
+<button id="search-button">search</button>
```

Comment:
_UI improvement_ : add the protection from submitting buttons when the user enters incorrect input data.

Reply:
fixed